Drawing.java

```diff
@@ -0,0 +1,51 @@
+package UnitOmega13;
+
+public class Drawing {
+	private char[][] part1;
+	private int maxSize;
+	public void drawStars(int maxNumberOfStars) {
+		this.maxSize = maxNumberOfStars;
+		if (maxSize%2 == 0) {
+			everySecLine();
+		}else {
+			everythirdLine();
```

Comment:
the same
https://mate-academy.github.io/style-guides/java/java.html#s5.1-identifier-names